### ISSUE TITLE
Workspace setup

### ISSUE THREAD

**Issue by stonier** [maliput/maliput_infrastructure#95]
TLDR; wsetup is a magic tool that masks workspace creation details / contains custom details.

* So much scripting/templating, it's not easy to determine what it *actually* does
* Should aim to become obsolete

What does that mean in terms of next steps? Just rough thoughts for now...

* A single command that can be called to ensure 'dsim build essentials' are configured & installed
* A single command that synchronises an underlay to a) built ament binaries and b) 3rd party non-ament binaries
* Do we need our own rosdep source (maybe other workarounds for non-deb/non-ament packages)

At that point we should have a very simple setup script that works with vcs, rosdep, colcon and instructions mostly revolve around instructions on those tools.
